morecantile/models.py

```diff
@@ -1389,21 +1389,24 @@ def feature(
                 UserWarning,
                 stacklevel=1,
             )
-            if epsg := feature_crs.to_epsg():
+            if authority_code := feature_crs.to_authority(min_confidence=20):
+                authority, code = authority_code
                 feat.update(
                     {
                         "crs": {
-                            "type": "EPSG",
-                            "properties": {"code": epsg},
+                            "type": "name",
+                            "properties": {
+                                "name": f"urn:ogc:def:crs:{authority}:0:{code}"
```

Comment:
hmmm, I think it would be safer to use the CRS_to_uri here, what if the crs is  CRS84? (result should be urn:ogc:def:crs:OGC:1.3:CRS84). Hopefully not huge difference

Reply:
👍 made the change